### ISSUE TITLE
Support for newer drafts?

### ISSUE THREAD

**Issue by handrews** [snowplow/schema-guru#178]
Hello- I'm dropping by from the JSON Schema spec team to notify this project that the current draft is draft-07, which is a minor update of the more significant draft-06.

[Guide for core and validation](http://json-schema.org/draft-07/json-schema-release-notes.html)
[Guide for hyper-schema](http://json-schema.org/draft-07/json-hyper-schema-release-notes.html)

Note that implementations that do not support **at least draft-06** will soon be moved to an "obsolete" or "older" section of the json-schema.org web site.  Implementations supporting draft-06 or newer will remain prominently featured on the existing implementations page.

For tools generating schemas that do not specify a draft, this would just mean not emitting any keywords that are incompatible with draft-06.

**Comment by alexanderdean** [comment]
Thanks for the heads-up @handrews !

**Comment by chuwy** [comment]
Yep, thanks for heads-up. This is pending for similar issue from Iglu (schema-guru's main consumer) https://github.com/snowplow/iglu/issues/303, but definitely on our radar
